### ISSUE TITLE
No loss.backward()

### ISSUE THREAD

**Issue by borisgribkov** [aalto-intelligent-robotics/sivl#2]
Dear author! First of all, thanks for your code! 
Checked it briefly I didn't found backward pass during training stage, it should be somewhere here https://github.com/aalto-intelligent-robotics/sivl/blob/main/code/trainSimilarityModel.py#L151-L174 I guess. Thanks!

**Comment by jkinnari** [comment]
Thank you for showing interest in this work and thank you for highlighting this. You are right, I had accidentally deleted that row when cleaning up code for publishing. Now it should find weights that lead to similar performance as the provided checkpoint.

**Comment by borisgribkov** [comment]
Dear @jkinnari! Thanks for your answer, clear now!
